homeassistant/components/media_player/sonos.py

```diff
@@ -608,6 +613,13 @@ def update(self):
             else:
                 playlist_size = int(playlist_size)
 
+            if playlist_size is None or playlist_size == 0:
```

Comment:
If we start catching 701 errors we wouldn't have to implement any business logic and just make sure we don't log them. That sounds more maintainable. Is there any added value from going down the business logic route?